packages/flutter_tools/test/src/fakes.dart

```diff
@@ -403,14 +400,7 @@ class ThrowingPollingDeviceDiscovery extends PollingDeviceDiscovery {
   bool get canListAnything => true;
 }
 
-class TestPlistParser extends PlistParser {
-  TestPlistParser()
-      : super(
-          fileSystem: MemoryFileSystem.test(),
-          logger: BufferLogger.test(),
-          processManager: FakeProcessManager.any(),
-        );
-
+class FakePlistParser extends Fake implements PlistParser {
```

Comment:
If you're not leaving APIs unimplemented you don't need to extend fake

Reply:
Until a new method is added to `PlistParser`. But I guess then the compiler would yell at me to implement it.

Reply:
Yeah, either the compiler warns you or some tests fail (hopefully)